content/en/developers/marketplace/offering.md

```diff
@@ -0,0 +1,262 @@
+- [Professional services](#build-a-bi-directional-data-integration)
+
+### Build an Agent-based integration
+
+{{< tabs >}}
+{{% tab "Datadog Agent-Based Integration" %}}
+
+An [OpenMetrics check][1] is suitable for gathering telemetry data from existing applications that expose metrics using the Open Metrics standard.
+
+
+[1]: https://docs.datadoghq.com/developers/custom_checks/prometheus/
+{{% /tab %}}
+{{% tab "Python Check" %}}
+
+A [Python check][1] is suitable for monitoring services or products that do not expose metrics in a standard format. The Python check is used to collect telemetry data from various APIs or command line tools.
+
+
+[1]: https://docs.datadoghq.com/developers/integrations/new_check_howto/?tab=configurationtemplate#write-the-check
+{{% /tab %}}
+{{% tab "DogStatsD" %}}
+
+[DogStatsD][1] is suitable for applications that already emit telemetry using the StatsD protocol. 
+
+Datadog adds additional Datadog-specific extensions to the StatsD protocol, including the following:
+
+- Histogram Metric Type
+- Service Checks
+- Events
+- Tagging
+
+
+[1]: https://docs.datadoghq.com/developers/dogstatsd/
+{{% /tab %}}
+{{< /tabs >}}
+
+For more information about Datadog Agent-based integrations, see [Introduction to Agent-based Integrations][3].
+
+### Build a REST API integration
+
+Use an [API integration][4] to enrich and submit data from your backend, or pull data directly out of Datadog. API integrations work well in building a connector between Datadog and another SaaS platform.
+ 
+Since API integrations do not use the Datadog Agent to collect data, you need to create an [informational tile-only listing](#) once your development work is complete.
+ 
+A Datadog API key is required to submit data to a Datadog API endpoint, and an application key is required to query data from Datadog or create resources on the Datadog site. Optionally, you can setup [OAuth for a data integration][5] in a Marketplace tile instead.
+
+### Build a Datadog App
+
+[Datadog Apps][6] are custom dashboard widgets that are developed in the [Datadog Developer Platform][7]. Once your Datadog App is ready to publish, you need to create an informational tile-only listing on the **Integrations** or **Marketplace** page.
+
+### Build an informational tile-only listing
+
+If you are developing an offering that does not use the Datadog Agent (such as a Datadog App, SaaS license, professional service, or an API-based integration) to collect data, you only need to create an informational tile listing. These tile-only listings require Datadog users to install and configure the integration outside of Datadog.
+
+The Datadog Development Toolkit offers a command option to create tile-only scaffolding: `ddev create -t tile -v2 "<Offering Name>"`. When using this command, you only receive the files related to your tile, instead of all the files used to build a full Agent-based data integration.
```

Comment:
```suggestion
The Datadog Development Toolkit offers a command option to create tile-only scaffolding: `ddev create -t tile "<Offering Name>"`. When using this command, you only receive the files related to your tile, instead of all the files used to build a full Agent-based data integration.
```
v2 is no longer needed